### ISSUE TITLE
Where is the Graph Enhancement Module implemented in the released code?

### ISSUE THREAD

**Issue by jeongHwarr** [yangzhifeio/MMGDreamer#11]
Hello authors,
Thank you for sharing your excellent work MMGDreamer.
I’ve been reviewing your paper and the released implementation, and I have a question regarding the Graph Enhancement Module described in Figure 2 (Section 3.2 of the paper).

In the paper, the Graph Enhancement Module consists of:

- Visual Enhancement Module (Encoder–Decoder + Codebook)
- Relation Predictor (GCN + MLP, trained with cross-entropy loss)

However, in the released code:

The main training script (train_3dfront_mask.py) directly calls

```
model = SGDiff(type='mmgdreamer', ...)
model.forward_mani(...)
```

which in turn wraps `Sg2ScDiffModel` from model/mmgdreamer.py.

Inside `Sg2ScDiffModel`, there are `GraphTripleConvNet`, `LayoutDiff`, and `ShapeDiff` modules, but I could not find any Visual Enhancement or Relation Predictor components.

Where is the Graph Enhancement Module implemented in the released code?

**Comment by yangzhifeio** [comment]
Hi, Thank you for your attention.
These two modules are trained separately, and the code has not yet been organized for open-source release.
